apps/nuxt3-ssr/tests/header-items.spec.ts

```diff
@@ -16,5 +16,12 @@ test("should show variables in menu if there are variables", async ({
   goto,
 }) => {
   await goto("/catalogue-demo/ssr-catalogue/all", { waitUntil: "hydration" });
-  await expect(page.getByRole("navigation")).toContainText("Variables");
+  await expect(page.getByRole("navigation")).toContainText("Resources");
+  await expect(page.getByRole("navigation")).toContainText("Cohort studies");
```

Comment:
Ideally WHEN more than one resource type has count>0 THEN I would like the 'cohort studies' etc underneath the 'resources' such that both 'resources' and 'variables' can be shown.